inetum-realjobs-frontend/src/app/components/resume/details/education-list/education-list.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {Education} from "../../../../models/education";
+
+@Component({
+  selector: 'app-education-list',
+  templateUrl: './education-list.component.html',
+  styleUrls: ['./education-list.component.scss']
+})
+export class EducationListComponent implements OnInit {
+
+  @Input() educationList: Education[];
+
+  @Input() deleteButton: boolean;
+  @Output() deleteEvent = new EventEmitter<number>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+  }
```

Comment:
Empty `ngOnInit` should be removed. Don't forget to remove `implements OnInit` for the class.